tools/tpp-opt/tpp-opt.cpp

```diff
@@ -30,10 +30,12 @@
 #include "TPP/Dialect/Xsmm/XsmmDialect.h"
 #include "TPP/PassBundles.h"
 #include "TPP/Passes.h"
+#include "TPP/Conversion/ConvertVectorToXsmm/ConvertVectorToXsmm.h"
 
 int main(int argc, char **argv) {
   mlir::registerAllPasses();
   mlir::tpp::registerTppCompilerPasses();
+  mlir::tpp::registerConvertVectorToXsmmPass();
```

Comment:
I'd expect the pass registration to be already included in `registerTppCompilerPasses`.
Is a separate one really needed?